test/utils/calculateBatchId.spec.ts

```diff
@@ -11,15 +13,15 @@ describe('calculateBatchId', () => {
     expect(calculateBatchId()).toBe(BATCH_ID)
   })
   it('returns next batch id when exact time is set for next batch', () => {
-    const fiveMinutesInTheFuture = addTimeToDate(DATE, 5, 'minute')
+    const fiveMinutesInTheFuture = addMinutes(DATE, 5)
```

Comment:
I feel we are testing their lib :) 
Maybe consider reexporting the `addMinutes` and so on in our utils so we test our helper and remove direct dependencies to 'date-fns'

Reply:
Not really, because `addMinutes` is just a helper for the test. `calculateBatchId` doesn't use `date-fns` at all.